### ISSUE TITLE
Normalize implant names in cloud db

### ISSUE THREAD

**Issue by bjhardcastle** [AllenInstitute/npc_sessions#86]
- should match names here https://github.com/AllenInstitute/npc_shields/blob/main/src/npc_shields/shields.py
- see drawings for ref https://github.com/AllenInstitute/npc_shields/tree/main/src/npc_shields/drawings

**Comment by arjunsridhar12345** [comment]
These should be good once the sync happens Sunday. A note below:

 Checked with Ethan and confirmed, a couple of the early templeton subjects (620263, 620264) used an older implant TS5

We can discuss later if needed depending on if that will break anything and such.

**Comment by bjhardcastle** [comment]
Thanks a lot for doing this. 'TS5' should be fine, as long as it's consistent.
________________________________

**Comment by bjhardcastle** [comment]
> a couple of the early templeton subjects (620263, 620264) used an older implant TS5
@arjunsridhar12345 I think I just understood this: you mean they're a version of TS5 that is not the same as the "official" TS5?

2002/TS5 seem to be synonymous, and we use `2002` in npc_shields. 

@egmcbride Should 620263 & 620264 be labelled something other than `2002`?

**Comment by arjunsridhar12345** [comment]
yeah i believe its an older version that follows this configuration: `\\allen\programs\mindscope\workgroups\dynamicrouting\dynamic_gating_insertions\implant_designs\TS5.PNG`

**Comment by bjhardcastle** [comment]
Ok thanks - I think that is the true TS5, and 2002 is not synonymous with TS5.

More info from Heston S a while back:
>•	0251-110-42, original concept for the DR implant. Referred to as ‘the football’ in the entries below. Also called the ‘42’ implant
•	0270-100-04, modified version of the 42 implant. Tamina did the mods, hence these are referred to as ‘Tamina Special #4’ or ‘TS-4’ 
•	0270-100-05, another modified 42 implant. ‘Tamina Special #5’ or ‘TS-5’ 
•	0283-200-001, Templeton Implant V.1. Requested by Ethan McBride, first implant made using our new production process
•	0283-200-002, I’m referring to this as ‘Dynamic Routing V.1’ since this is slated to be the first production Dynamic Routing Implant. The one with a bunch of holes.
